Content.Server/Storage/EntitySystems/StorageSystem.Fill.cs

```diff
@@ -39,8 +50,42 @@ private void OnStorageFillMapInit(EntityUid uid, StorageFillComponent component,
                 Logger.ErrorS("storage", $"Tried to StorageFill {entry.PrototypeId} inside {uid} but can't.");
                 EntityManager.DeleteEntity(ent);
             }
+        }
+
+        // handle orgroup spawns
+        foreach (var group in orGroupedSpawns)
+        {
+            double diceRoll = _random.NextDouble() * group.Value.CumulativeProbability;
+            _random.Shuffle(group.Value.Entries);
```

Comment:
The shuffle is unnecessary

Reply:
I'm no statistician, I just assumed that items at the top of the list might have a higher chance to spawn in theory than lower ones, but will remove.